pkgs/applications/graphics/awesomebump/default.nix

```
@@ -2,12 +2,12 @@
 
 
 let
-  version = "5.1";
+  version = "5.1.1";
 
   src = fetchgit {
     url = "https://github.com/kmkolasinski/AwesomeBump.git";
```

Prefer `fetchFromGitHub` to `fetchgit`:

```suggestion
  src = fetchFromGitHub {
    owner = "kmkolasinski";
    repo = "AwesomeBump";
```

(And don't forget to replace `fetchgit` with `fetchFromGitHub` on line 1, but GitHub won't let me suggest  a change on line 1 for some reason...)

---

It seem to fail. I'm sure that something related to submodule (if I'm okay, fetchfromgithub fetch the tarball directly from github, but those tarball doesn't have submodule embedded).